app/validators/admin/csv/facility_validator.rb

```diff
@@ -29,14 +29,22 @@ def duplicate_rows
   def facilities
     row_errors = []
     @facilities.each.with_index(2) do |facility, row_num|
-      import_facility = Facility.new(facility)
+      import_facility = Facility.new(sanitize_facility(facility))
       row_errors << [row_num, import_facility.errors.full_messages.to_sentence] if import_facility.invalid?
     end
     group_row_errors(row_errors).each { |error| @errors << error } if row_errors.present?
   end
 
   private
 
+  def sanitize_facility(facility)
```

Comment:
This isn't really sanitizing as much as parameter massaging. Can we name this `facility_params` or `facility_attrs` or something?

I'm guessing this is the first case where columns in the import spreadsheet don't map exactly to columns in our db?

Reply:
Yup